src/components/header/Index.jsx

```diff
@@ -1,20 +1,26 @@
-/* eslint-disable no-unused-expressions */
 /* eslint-disable react/jsx-no-bind */
+/* eslint-disable react/jsx-indent */
```

Comment:
@olajide1234 we're disabling indenting too?

Reply:
This rule is now retained.